src/hotspot/share/gc/z/zLargePages.cpp

```diff
@@ -30,8 +30,11 @@ bool ZLargePages::_os_enforced_transparent_mode;
 
 void ZLargePages::initialize() {
   pd_initialize();
-
-  log_info_p(gc, init)("Memory: " JULONG_FORMAT "M", os::physical_memory() / M);
+  size_t phys_mem = 0;
+  if (!os::physical_memory(phys_mem)) {
+    log_debug_p(gc, init)("os::physical_memory() failed");
+  }
+  log_info_p(gc, init)("Memory: %zuM", phys_mem / M);
```

Comment:
I have the same comment about this change as I have for `GCInitLogger::print_memory`.